AAInstrumentation/aa_reduction.py

```diff
@@ -3,6 +3,7 @@
 from dataclasses import dataclass
 from pathlib import Path
 import sys
+from abc import abstractmethod
```

Comment:
unused?

Reply:
It was used before, which actually made the earlier PR not work at all :sweat_smile: 
So essentially, I am silently fixing my mistakes here.